CHANGELOG.md

```diff
@@ -3,3 +3,8 @@
 ## v1.0.0
 
 Initial public release for terraform-provider-chronosphere.
+
+## Unreleased
```

Comment:
can you move this to above the v1.0.0 (we generally do latest release at the top)